final_task/rss_reader/printers.py

```diff
@@ -0,0 +1,111 @@
+from collections import OrderedDict
+from collections import defaultdict
+import logging as log
+import json
+import rss_get_items as filters
+import pprint
+from colorama import Fore
+from colorama import Style
+
+
+def split_string_by_lines(input_string: str, word_number: int) -> str:
+    input_string = input_string.strip().split()
+    result = ''
+    for idx in range(len(input_string) // word_number + 1):
+        start = idx * word_number
+        end = (idx + 1) * word_number
+        result += ' '.join(input_string[start: end]) + '\n'
+    return result.strip()
+
+
+def prepare_one_item(item_xml: defaultdict) -> OrderedDict:
+    """"
+
+    Take one rss item as dictionary and make ordered dict
+    with title, date, description and media content
+
+    """
+    title = item_xml['title']
+    date = filters.pubdate(item_xml['pubDate'])
+    date = data_split(date)
+    news_link = item_xml['link']
+    description = split_string_by_lines(
+        filters.description(item_xml['description']),
+        10
+    )
+    media_content = split_string_by_lines(
+        ' '.join(item_xml['content']),
+        1
+    )
+    prepared_news = OrderedDict()
```

Comment:
In python version 3.7+ dictionary is ordered by default.

Reply:
I know this, but if someone accidentally uses python3.6 for example, it will not be good if he has undefined behavior.